app/main.py

```diff
@@ -16,4 +16,8 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, profile_info, marks],
+    "immutable": [lucky_number,pi, one_is_a_prime_number, name, collection_of_coins]
```

Comment:
The variable `collection_of_coins` is a set, which is mutable. It should be placed in the 'mutable' list instead of 'immutable'.